.travis.yml

```diff
@@ -2,18 +2,30 @@ language: python
 dist: xenial
 services:
 - docker
-matrix:
+
+install:
+- pip install tox
+
+script:
+- tox
+
+jobs:
   include:
-  - python: 2.7
-    env: TOXENV=py27
-  - python: 2.7
-    env: TOXENV=py27-functional
-  - python: 2.7
+  - stage: verify-tag
+    python: 3.7
+    script: |
+      [ "v$(python -c 'import kubernetes ; print(kubernetes.__version__)')" == "${TRAVIS_TAG}" ]
```

Comment:
We could also put in a check to make sure that the `TRAVIS_TAG` obeys semver

Reply:
I was going to say that. This would, in turn, ensure that the version that we put in constants.py is also validated.